src/Trixi.jl

```diff
@@ -150,8 +150,9 @@ export flux, flux_central, flux_lax_friedrichs, flux_hll, flux_hllc, flux_hlle,
        flux_chandrashekar, flux_ranocha, flux_derigs_etal, flux_hindenlang_gassner,
        flux_nonconservative_powell,
        flux_kennedy_gruber, flux_shima_etal, flux_ec,
-       flux_fjordholm_etal, flux_nonconservative_fjordholm_etal, flux_es_fjordholm_etal,
```

Comment:
Unfortunately, we still do. The LLF-dissipation is now added to `flux_wintermeyer_etal ` instead of `flux_fjordholm_etal` and the resulting ES-flux was renamed to `flux_es_ersing_etal`.